tempus_fugit_minecraft/main.py

```diff
@@ -430,6 +438,34 @@ def process_entire_queue(self):
         """
         while self.queue:
             self._dequeue()
+    
+    
+    def generate_clouds(self,n,num_of_clouds = 250):
+        """
+        Generate the position of the clouds on the sky.
+        
+        Inputs: n = 1/2 size of the world.
+                num_of_clouds = default clouds to be generated =250
+        
+        Output: return a list of lists; each inner list represents a set of cloud blocks.
+        """
+        o = n - 10
+        clouds = list()
+        for _ in xrange(num_of_clouds):
+            a = random.randint(-o, o)  # x position of the cloud
+            b = random.randint(-o, o)  # z position of the cloud
+            c = 20                     # y position of the cloud (height)
```

Comment:
why not name these variables x, y, z instead of a, b, c? might be better for code readability

Reply:
you are right. except that `a`,`b`,`c` here represent the center of a single cloud that the cloud blocks will be surrounding this center. In the nested for loop, we are actually run through each block's coordinates and here we call them `x`,`y`(or `c` -- represents the height of the cloud),`z`. those coordinates belong to a single block.

I would happily do the changes it but on the other hand, I need to change the variables on the nested loops, which (for me) makes more sense to keep.